src/Pos/Types/Arbitrary/Core.hs

```diff
@@ -37,3 +39,21 @@ instance Arbitrary Types.EpochOrSlot where
           Types.EpochOrSlot . Left <$> arbitrary
         , Types.EpochOrSlot . Right <$> arbitrary
         ]
+
+deriving instance Arbitrary Fee.Coeff
+
+instance Arbitrary Fee.TxSizeLinear where
+    arbitrary = genericArbitrary
+    shrink = genericShrink
+
+instance Arbitrary Fee.TxFeePolicy where
+    arbitrary = do
+        v <- arbitrary
+        case v of
```

Comment:
I don't like it. It will almost always be `Unknown`. I think probabilities of `TxSizeLinear` and `Unknown` should be similar.